builtin/logical/database/backend.go

```diff
@@ -24,6 +25,7 @@ const (
 	databaseConfigPath     = "database/config/"
 	databaseRolePath       = "role/"
 	databaseStaticRolePath = "static-role/"
+	minRootCredRollbackAge = 1 * time.Minute
```

Comment:
The reason I set this to 1 minute is because of the suggestion on the field comment [backend.go#L74](https://github.com/hashicorp/vault/blob/master/sdk/framework/backend.go#L74). I observed that the context associated with the rotate root request expires after 1 minute.

With the min age at 10 seconds, I worry that it may be possible to process a WAL entry for a rotate root request that is potentially still in-flight. For example, this could happen if the rotate root request takes longer than 10 seconds (hanging client, perhaps) but would eventually succeed. Thoughts?

Reply:
Ah! Gotcha! That makes total sense. Thanks for the explanation.